### ISSUE TITLE
Updating to Chromium 88.0.4324.150

### ISSUE THREAD

**Issue by Zoraver** [Eloston/ungoogled-chromium#1379]
## Notes for the developer

Once you claim it, it is advisable that you create a [Draft Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) for increased visibility:

![GitHub Interface for creating Draft Pull Requests](https://help.github.com/assets/images/help/pull_requests/pullrequest-send.png)

Finally, make sure to reference this issue in your PR.

Feel free to ask questions and discuss issues here along the way. Thanks!

## Notes for others

Feel free to raise issues or questions throughout the process. However, please refrain from asking ETAs unless no visible progress has been made here or on the developer's PR for a while (e.g. 2 weeks).

**Comment by Zoraver** [comment]
I plan on working on this; Chromium 88.0.4324.150 fixes a security vulnerability which Google believes to be actively exploited in the wild.

**Comment by Zoraver** [comment]
I just wanted to give you guys an update; I am still waiting for Google to post the source tarball for Chromium 88.0.4324.150...

**Comment by kramred** [comment]
There were no changes in pdfium or skia from `.146` to `.150`, some in [chromium](https://chromium.googlesource.com/chromium/src.git/+log/1cc23a5452e78b11e16725838c341efb8cb818fd..2489b0741310c3a50c990c1d6acb9f113bd879a8?n=1000&pretty=fuller) (not relevant to security), and the security relevant one in [v8](https://chromium.googlesource.com/v8/v8.git/+log/e71858f7a080b9bd813876c13be4ce30d993d1d9..7439c236cb1282f351ee5fdec0b4b871d9606aee?n=1000&pretty=fuller). I got the `v8` patchset from [Chromium Gerrit](https://chromium-review.googlesource.com/c/v8/v8/+/2675930) and applied it to `.146` – all ungoogled chromium patches passed and I'm building now on my build system (for x86-64 macos).

TL;DR: no adjustments to ungoogled chromium patches should be necessary

**Comment by wchen342** [comment]
Checked on Android version. Can confirm the patches used here are not affected.
